dlc-wallet/src/main/resources/postgresql/dlc/migration/V1__dlc_db_baseline.sql

```diff
@@ -1,15 +1,15 @@
-CREATE INDEX "wallet_dlc_accepts_event_id_index" on "wallet_dlc_accepts" ("event_id");
+CREATE TABLE "wallet_dlc_accepts" ("param_hash" VARCHAR(254) NOT NULL UNIQUE,"funding_key" TEXT NOT NULL,"payout_address" TEXT NOT NULL,"total_collateral" INTEGER NOT NULL,"change_address" TEXT NOT NULL,constraint "pk_dlc_accept" primary key("param_hash"), constraint "fk_param_hash" foreign key("param_hash") references "wallet_dlcs"("param_hash") on update NO ACTION on delete NO ACTION);
+CREATE INDEX "wallet_dlc_accepts_param_hash_index" on "wallet_dlc_accepts" ("param_hash");
 
-CREATE TABLE "wallet_dlc_funding_inputs" ("event_id" VARCHAR(254) NOT NULL UNIQUE,"is_initiator" BOOLEAN NOT NULL,"out_point" TEXT NOT NULL UNIQUE,"output" TEXT NOT NULL,"redeem_script_opt" TEXT, "witness_script_opt" TEXT,"sigs" TEXT,constraint "pk_dlc_input" primary key("out_point"), constraint "fk_event_id" foreign key("event_id") references "wallet_dlcs"("event_id") on update NO ACTION on delete NO ACTION);
+CREATE TABLE "wallet_dlc_funding_inputs" ("param_hash" VARCHAR(254) NOT NULL UNIQUE,"is_initiator" BOOLEAN NOT NULL,"out_point" TEXT NOT NULL UNIQUE,"output" TEXT NOT NULL,"redeem_script_opt" TEXT, "witness_script_opt" TEXT,"sigs" TEXT,constraint "pk_dlc_input" primary key("out_point"), constraint "fk_param_hash" foreign key("param_hash") references "wallet_dlcs"("param_hash") on update NO ACTION on delete NO ACTION);
 
-CREATE TABLE "wallet_dlc_cet_sigs" ("event_id" TEXT NOT NULL,"outcome_hash" TEXT NOT NULL,"signature" TEXT NOT NULL,constraint "pk_dlc_cet_sigs" primary key("event_id","outcome_hash"),constraint "fk_event_id" foreign key("event_id") references "wallet_dlcs"("event_id") on update NO ACTION on delete NO ACTION);
+CREATE TABLE "wallet_dlc_cet_sigs" ("param_hash" TEXT NOT NULL,"outcome_hash" TEXT NOT NULL,"signature" TEXT NOT NULL,constraint "pk_dlc_cet_sigs" primary key("param_hash","outcome_hash"),constraint "fk_param_hash" foreign key("param_hash") references "wallet_dlcs"("param_hash") on update NO ACTION on delete NO ACTION);
 
-CREATE TABLE "wallet_dlc_refund_sigs" ("event_id" TEXT NOT NULL,"is_initiator" INTEGER NOT NULL,"refund_sig" TEXT NOT NULL,constraint "fk_event_id" foreign key("event_id") references "wallet_dlcs"("event_id") on update NO ACTION on delete NO ACTION);
-CREATE INDEX "wallet_dlc_refund_sigs_event_id_index" on "wallet_dlc_accepts" ("event_id");
+CREATE TABLE "wallet_dlc_refund_sigs" ("param_hash" TEXT NOT NULL,"is_initiator" INTEGER NOT NULL,"refund_sig" TEXT NOT NULL,constraint "fk_param_hash" foreign key("param_hash") references "wallet_dlcs"("param_hash") on update NO ACTION on delete NO ACTION);
+CREATE INDEX "wallet_dlc_refund_sigs_param_hash_index" on "wallet_dlc_accepts" ("param_hash");
```

Comment:
Can this not be done in a new migration? I thought that was the point of moving dlc stuff to its own db?

Reply:
Can't really do it as a migration because old dlcs won't have a `temp_contract_id` or `contract_id`. From my understanding we moved it to its own db so we don't get conflicts with master for the wallet migrations

Reply:
I guess we could do it as a migration, but it won't be backwards compatible anyways so there is no real point